index.d.ts

```diff
@@ -0,0 +1,99 @@
+/*~ Note that ES6 modules cannot directly export class objects.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('autonumeric');
+ *~
+ *~ Alternatively, if --allowSyntheticDefaultImports or
+ *~ --esModuleInterop is turned on, this file can also be
+ *~ imported as a default import:
+ *~   import x from 'autonumeric';
+ *~
+ *~ Refer to the TypeScript documentation at
+ *~ https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require
+ *~ to understand common workarounds for this limitation of ES6 modules.
+ */
+ 
+/*~ If this module is a UMD module that exposes a global variable 'myClassLib' when
+ *~ loaded outside a module loader environment, declare that global here:
+ *~ export as namespace myClassLib;
+ */
+
+/* The class constructor function is the exported object from the file */
+export = AutoNumeric;
+
+declare class AutoNumeric {
+  constructor(
+    elementOrSelector: string | HTMLInputElement | HTMLElement,
+    defaultValue: string | number = null,
+    options: AutoNumericOptions | string = null
+  ): Input;
+
+  // someProperty: string[];
+
+  multiple(
+    elementsOrSelector:
+      | string
+      | HTMLElement[]
+      | { rootElement: HTMLElement; exclude?: HTMLInputElement[] },
+    initialValue: number | Array<number | null> = null,
+    options: AutoNumericOptions | AutoNumericOptions[] = null
+  ): Input;
+
+  /**
+   * Set the value, but do not save the new state in the history table (used for undo/redo actions)
+   */
+  set(
+    value: number | string | null,
+    options?: AutoNumericOptions,
+    save?: boolean
+  ): void;
+
+  update(...options: AutoNumericOptions[]): void;
+
+  remove(): void;
+
+  getNumber(
+    callback: (value: number | null, instance: AutoNumeric) => void = null
+  ): number | null;
+
+  getNumericString(
+    callback: (value: string | null, instance: AutoNumeric) => void = null
+  ): string | null;
+
+  static getPredefinedOptions(): AutoNumericOptions;
+}
```

Comment:
It misses the function definitions from #531 (ie. `version()`, `isManagedByAutoNumeric()`, etc.)

Reply:
See comment below